### PR TITLE
Remove duplicate `@skyux/auth-client-factory`

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**
`@skyux/auth-client-factory` is already installed by `@skyux/http`:
https://github.com/blackbaud/skyux-http/blob/master/package.json#L31

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -64,7 +64,6 @@
     "@ngtools/webpack": "7.3.9",
     "@pact-foundation/karma-pact": "2.2.0",
     "@pact-foundation/pact-node": "9.0.0",
-    "@skyux/auth-client-factory": "1.1.0",
```
Comment (Blackbaud-BobbyEarl) [inline package.json]: Is something else providing it?

Comment (Blackbaud-SteveBrush) [reply package.json]: Yes, see the pull request's description. ;-)

Comment (Blackbaud-BobbyEarl) [reply package.json]: I did not.  :-)

**Comment by Blackbaud-SteveBrush** [comment]
Closing in favor of: https://github.com/blackbaud/skyux-sdk-builder/pull/170

**Pull request closed without merging**